sodetlib/stream.py

```diff
@@ -207,6 +208,9 @@ def stream_g3_on(S, make_freq_mask=False, emulator=False, tag=None,
     S.set_downsample_factor(downsample_factor)
     S.set_filter_disable(int(filter_disable))
 
+    # ensure fixed tones are on
+    uxm_setup.turn_on_fixed_tones(S, cfg)
```

Comment:
Can we check `cfg.dev.bands[b]['fixed_tones']['enabled']` and only run this if True?

Reply:
it checks this band-by-band within `turn_on_fixed_tones`. We could check it here as well, but the result would be the same.